test/boulder-tools/Dockerfile

```diff
@@ -28,5 +31,6 @@ RUN sed -i '/$RepeatedMsgReduction on/s/^/#/' /etc/rsyslog.conf
 
 COPY --from=godeps /usr/local/bin/* /usr/local/bin/
 COPY --from=godeps /usr/local/go/ /usr/local/go/
+COPY --from=rustdeps /usr/local/cargo/bin/typos /usr/local/bin/typos
 
-ENV PATH /usr/local/go/bin:/usr/local/protoc/bin:$PATH
+ENV PATH /usr/local/go/bin:/usr/local/protoc/bin:/usr/local/bin:$PATH
```

Comment:
You can remove this change because this directory is already on the PATH.
```
$ docker run -it letsencrypt/boulder-tools bash -c 'echo ${PATH}'
/usr/local/go/bin:/usr/local/protoc/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
Done.